webpack.config.js

```diff
@@ -95,14 +96,12 @@ const baseConfig = {
 
 /**@type {import('webpack').Configuration}*/
 const webviewConfig = {
+    name: 'vueLoader',
```

Comment:
What was the reason again for needing to run the builds separate? I'm fine with this solution, just curious why it wasn't working before.

Reply:
Two reasons:
* Vue (and all frontend frameworks) need to be webpacked so this gives us a faster turnaround while developing; these aren't quite a "part" of the extension, they're separate HTML
* The other webpack webpacks for Node use instead of this which webpacks for frontend.